assets/src/edit-story/app/story/actions/useSaveStory.js

```diff
@@ -90,6 +100,16 @@ function useSaveStory({ storyId, pages, story, updateStory }) {
       publisherLogo,
     } = story;
 
+    const formattedTitle = titleFormatted(title);
+
+    if (!formattedTitle || formattedTitle === '') {
+      showSnackbar({
+        message: __('Title is required to save story', 'web-stories'),
+      });
+      return;
+    }
```

Comment:
For me it should be more like (clear intention what we are doing here + titleFormatted logic inside this validator):
```js
if (!isValidTitle(title)) {
  // show error
}
```

Reply:
Go with disable button